collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -380,7 +380,11 @@ public void onCreate(Bundle savedInstanceState) {
         // Workaround for https://issuetracker.google.com/issues/37124582. Some widgets trigger
         // this issue by including WebViews
         if (Build.VERSION.SDK_INT >= 24) {
-            new WebView(this);
+            try {
+                new WebView(this);
+            } catch (Exception | Error e) {
+                // ignored
```

Comment:
Good to have a comment here like "Don't crash if `WebView` not available"

Reply:
Done.